examples/nx-example/README.md

```diff
@@ -2,6 +2,8 @@
 
 This repository was automatically created by running `npx create-nx-workspace@15.0.13 nx-example --preset angular-nest --nxCloud false --appName nx-example` and choosing SCSS as a default stylesheet.
 
-## Steps to follow
+## Steps to setup this repo yourself
+Add the dagger dev dependency by running
+`$ npm i -D @dagger.io/nodejs-sdk`
```

Comment:
Update:
the package name will be publish as `@dagger.io/dagger`

Reply:
https://github.com/dagger/dagger/blob/61fce278f264820522aa061098b33a4231eac972/sdk/nodejs/package.json#L2